sdk/src/Services/AppRegistry/Properties/AssemblyInfo.cs

```diff
@@ -15,6 +15,8 @@
 [assembly: AssemblyDescription("The Amazon Web Services SDK for .NET (NetStandard 2.0) - AWS Service Catalog App Registry. AWS Service Catalog AppRegistry provides a repository of your applications, their resources, and the application metadata that you use within your enterprise.")]
 #elif NETCOREAPP3_1
 [assembly: AssemblyDescription("The Amazon Web Services SDK for .NET (.NET Core 3.1) - AWS Service Catalog App Registry. AWS Service Catalog AppRegistry provides a repository of your applications, their resources, and the application metadata that you use within your enterprise.")]
+#elif NET6_0
+[assembly: AssemblyDescription("The Amazon Web Services SDK for .NET (6.0) - AWS Service Catalog App Registry. AWS Service Catalog AppRegistry provides a repository of your applications, their resources, and the application metadata that you use within your enterprise.")]
```

Comment:
💡 These would likely be simplified tremendously if a Directory.Build.props/Directory.Build.targets file was used to generate them from a template across all projects. Would be a follow-up and not part of this pull request.

Reply:
Another great callout.  Agree w/ not adding it to the context of this PR, and it would make this much easier.